### PR TITLE
Hotfix/pmc

### PR DESCRIPTION
**Pull request by RJSheppard**
Fixing PMC timesteps bug

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# malariasimulation 2.1.0
+
+* Fix PMC bug where timesteps to generate targetted individuals were not increasing
```
Comment (pwinskill) [inline NEWS.md:3]:
```suggestion
* Fix PMC bug where timesteps to generate targeted individuals were not increasing
```

**Review by pwinskill — APPROVED** [review]
👍👍

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: malariasimulation
 Title: An individual based model for malaria
-Version: 2.0.0
+Version: 2.1.0
```
Comment (giovannic) [inline DESCRIPTION:3]:
```suggestion
Version: 2.0.1
```

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# malariasimulation 2.1.0
```
Comment (giovannic) [inline NEWS.md:1]:
```suggestion
# malariasimulation 2.0.1
```

File: DESCRIPTION (hunk 2 above)
Comment (giovannic) [reply DESCRIPTION:3]: Changing to 2.0.1 to signify a bug fix with no breaking changes! [Semver](https://semver.org/spec/v2.0.0.html) for reference

**Review by giovannic — CHANGES_REQUESTED** [review]

**Review by giovannic — APPROVED** [review]

**Pull request merged**